iot/iot-tenant-cleaner/src/main/java/io/enmasse/iot/tools/cleanup/InfinispanTenantCleaner/InfinispanConfigValues.java

```diff
@@ -0,0 +1,115 @@
+/*
+ *  Copyright 2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.enmasse.iot.tools.cleanup.InfinispanTenantCleaner;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.ArrayList;
+
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class InfinispanConfigValues {
+
+    @JsonProperty("iotProject")
+    private String tenantIdToClean;
+
+    private String host;
+    private int port = 11222;
+    private String username;
+    private String password;
+
+    @JsonProperty("saslServerName")
+    private String saslServer;
+    private String saslRealm;
+
+    public String getTenantIdToClean() {
+        return tenantIdToClean;
+    }
+
+    public void setTenantIdToClean(String tenantIdToClean) {
+        this.tenantIdToClean = tenantIdToClean;
+    }
+
+    public String getHost() {
+        return host;
+    }
+
+    public void setHost(String host) {
+        this.host = host;
+    }
+
+    public int getPort() {
+        return port;
+    }
+
+    public void setPort(int port) {
+        this.port = port;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public String getSaslServer() {
+        return saslServer;
+    }
+
+    public void setSaslServer(String saslServer) {
+        this.saslServer = saslServer;
+    }
+
+    public String getSaslRealm() {
+        return saslRealm;
+    }
+
+    public void setSaslRealm(String saslRealm) {
+        this.saslRealm = saslRealm;
+    }
+
+    public String verify(){
+
+        ArrayList<String> missingValues = new ArrayList<>();
+
+        if (tenantIdToClean == null) {
+            missingValues.add("iotProject ");
+        }
+        if (host == null) {
+            missingValues.add("host");
+        }
+        if (username == null) {
+            missingValues.add("username");
+        }
+        if (password == null) {
+            missingValues.add("password");
+        }
+        if (saslServer == null) {
+            missingValues.add("saslServer");
+        }
+        if (saslRealm == null) {
+            missingValues.add("saslRealm");
+        }
+
+        if (missingValues.size() != 0){
+            StringBuilder sb = new StringBuilder("Missing configuration value(s) : ");
```

Comment:
You should be able to use `java.lang.String.join(CharSequence, Iterable<? extends CharSequence>)`instead